apps/callscreen/js/calls_handler.js

```diff
@@ -295,12 +295,23 @@ var CallsHandler = (function callsHandler() {
       if (contact && contact.name) {
         CallScreen.incomingInfo.classList.add('additionalInfo');
         CallScreen.incomingNumber.textContent = contact.name;
-        CallScreen.incomingNumberAdditionalTelType.textContent =
-          Utils.getPhoneNumberAdditionalInfo(matchingTel);
+
+        var additionalL10n =
+          Utils.getLocalizedPhoneNumberAdditionalInfo(matchingTel);
+        if (additionalL10n.id) {
+          navigator.mozL10n.setAttributes(
+            CallScreen.incomingNumberAdditionalTelType,
+            additionalL10n.id,
+            additionalL10n.args);
+        } else {
+          CallScreen.incomingNumberAdditionalTelType.setAttribute(
```

Comment:
On second thought we might also do another small change: make Utils.getLocalizedPhoneNumberAdditionalInfo() always return an object with an id and args field even when only the id is used. Then we'd be able to just do:
```javascript
  var additionalL10n = Utils.getLocalizedPhoneNumberAdditionalInfo(matchingTel);
  navigator.mozL10n.setAttribute(
    CallScreen.incomingNumberAdditionalTelType,
    additionalL10n.id,
    additionalL10n.args
  );
```
Look ma', no if()s!